modules/virt-removing-wasp-agent.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * virt/post_installation_configuration/virt-configuring-higher-vm-workload-density.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-removing-wasp-agent_{context}"]
+= Removing wasp-agent
+
+You can remove the `wasp-agent` component and associated resources from your cluster if you no longer need memory overcommitment.
+
+.Prerequisites
+
+* You are logged in to the cluster with the `cluster-admin` role.
+* You installed the OpenShift CLI (`oc`).

```

Comment:
The `common-attributes` file has an attribute called `oc-first` for "OpenShift CLI (`oc`)". Consider using it to keep consistency among our docs.